java-engine/src/main/java/io/getunleash/engine/CustomStrategiesEvaluator.java

```diff
@@ -61,10 +67,13 @@ public void loadStrategiesFor(String toggles) {
     }
   }
 
-  private List<MappedStrategy> getFeatureStrategies(FeatureDefinition feature) {
+  List<MappedStrategy> getFeatureStrategies(FeatureDefinition feature) {
     List<MappedStrategy> mappedStrategies = new ArrayList<>();
     int index = 1;
     for (StrategyDefinition strategyDefinition : feature.strategies) {
+      if (builtinStrategies.contains(strategyDefinition.name)) {
```

Comment:
This check is actually the salient point here. If we find our strategy matches a built in strategy, just continue